MonkeyBot/Database/MonkeyDBContext.cs

```diff
@@ -21,13 +21,15 @@ public class MonkeyDBContext : DbContext
         public DbSet<GameSubscription> GameSubscriptions { get; set; }
         public DbSet<RoleButtonLink> RoleButtonLinks { get; set; }
 
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
```

Comment:
I didn't want to mess with making the EF context DbSet's nullable - nor handle nullability anywhere it's used, since I know it'll be assigned by EF. Instead I just disabled the warning for this lone instance.

Reply:
good catch